cypress/integration/User_can_log_into_SmartFAQ.js

```diff
@@ -0,0 +1,21 @@
+// @noflow
+/* global
+  Cypress,
+  cy
+*/
+
+describe('User can log into SmartFAQ', () => {
+  beforeEach(() => {
+    cy.visit('/');
+    cy.get('[data-cy=btn-existent-booking]').click();
+    cy.get('[data-cy=link-kiwi-login]').click();
+  });
+
+  it(`The user should be able to log into SmartFAQ with the correct credentials.`, () => {
+    cy.get('[data-cy=input-email]').type(Cypress.env('TEST_USER_EMAIL'));
+    cy.get('[data-cy=input-password]').type(Cypress.env('TEST_USER_PASSWORD'));
+    cy.get('[data-cy=btn-sign-in]').click();
+    cy.wait(1000);
```

Comment:
I dont think this is a good idea, because it could happen that it fails randomly.
But then again Im not sure if we should mock login functionality..
What do you think @git-toni @jaroslav-kubicek

Reply:
Yes, agree!

But... I don't know how to do without a mock. 
We need to find a better approach to do this.

Ideas? @git-toni @jaroslav-kubicek 

Thanks, @dinodsaurus !